src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -15,6 +15,7 @@
     public static final String CASH_REVERSION_OBJECT_CD = "CASH_REVERSION_OBJECT_CD";
     public static final String RULE_CODE_CA = "CA";
     public static final String DELIMITER = ".";
+    public static final String US_LOCALE_STRING = "en_US";
```

Comment:
This constant is for cases when we need to reference the US locale in an annotation parameter. With parameters on Java annotations, their values have to be constant expressions (like a literal string or a concatenation of literal strings), otherwise you will get a compiler error. Thus, Locale.US cannot be used directly as an annotation parameter value, even if written to a final variable first.